frontend/model/notifications/templates.js

```diff
@@ -207,6 +207,11 @@ export default ({
       name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${creatorID}`
     }
 
+    if (options.memberID) {
+      // NOTE: update member only when the type is  PROPOSAL_REMOVE_MEMBER
```

Comment:
This comment doesn't make sense.

There is no check for `PROPOSAL_REMOVE_MEMBER`. Please either update the comment or update the code so that it fits the comment.